README.md

```diff
@@ -1,37 +1,29 @@
-Terraform Provider for Google Cloud Platform
-==================
+# Terraform Provider for Google Cloud Platform
 
-- Website: https://www.terraform.io
+- Tutorials: [learn.hashicorp.com](https://learn.hashicorp.com/terraform?track=getting-started#getting-started)
+- Forum: [discuss.hashicorp.com](https://discuss.hashicorp.com/c/terraform-providers/tf-google/)
 - Documentation: https://www.terraform.io/docs/providers/google/index.html
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
-Maintainers
------------
+The Terraform Google provider is a plugin that allows [Terraform](https://www.terraform.io) to manage resources on Google Cloud Platform. This provider is maintained by the [Terraform team at Google](https://cloudplatform.googleblog.com/2017/03/partnering-on-open-source-Google-and-HashiCorp-engineers-on-managing-GCP-infrastructure.html) and the Terraform team at [HashiCorp](https://www.hashicorp.com/)
 
-This provider plugin is maintained by:
+The ['google-beta' provider](https://github.com/hashicorp/terraform-provider-google-beta) contains preview features and features at a beta [launch stage](https://cloud.google.com/products#product-launch-stages). Refer to the [provider versions](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_versions) documentation for more information about how to use `google-beta`.
```

Comment:
Maybe something like "This is the `google` provider, containing generally available features. To use preview features or features at a beta launch stage, you may use the `google-beta` provider..." here?